src/fmeval/eval_algorithms/common.py

```diff
@@ -64,6 +65,18 @@ def save_dataset(dataset: Dataset, score_names: List[str], save_strategy: SaveSt
                 save_strategy.save(batch["record"])
 
 
+def get_default_judge_model() -> BedrockModelRunner:
```

Comment:
Upon further reflection, I think that we should just remove this function entirely. Since a judge model is required for your eval algo, the user should explicitly provide it. Consider the case where the user doesn't even have access to AWS Bedrock. Then, the default model won't even work out of the box for them. Forcing them to provide their own judge model runner when calling `evaluate` or `evaluate_sample` removes this possibility, and makes for a less confusing experience overall, in my opinion.

Reply:
I agree. Is having a default judge model necessary? If yes, does it have to be a Bedrock model?